### ISSUE TITLE
Markdown export broken after upgrade

### ISSUE THREAD

**Issue by Gredin67** [YunoHost-Apps/etherpad_mypads_ynh#99]
Since a recent update of our etherpad instance we realized that the markdown export is broken: https://pad.sans-nuage.fr/p/cr/export/markdown
Is this issue known/reproducible?

**Comment by maniackcrudelis** [comment]
According to https://github.com/ether/ep_markdown/issues/10, an update of the plugin did break the export.

To update to Etherpad 1.8.0 solves that issue, just tried on a VM.
But, the version 1.8.0 is currently only available from the testing branch.

If you wish to upgrade nonetheless to that version to solve your issue, please do
```
sudo yunohost app upgrade etherpad_mypads -u https://github.com/YunoHost-Apps/etherpad_mypads_ynh/tree/testing
```

Otherwise, I hope to merge that branch in stable in a week.

**Comment by Gredin67** [comment]
Ok thanks for the fast feedback! Next week would be awesome!

**Comment by Jaxom99** [comment]
Hi, thanks for the answer and proposed timeline. Is this still in the works ? I have the same trouble on my server (ynh 3.8, non-testing version of the app).

**Comment by zamentur** [comment]
I close it because this issue is solved by current 1.8.0 (but there are some others issues)
